src/CoreWCF.Queue/src/CoreWCF/Queue/Common/QueueMessageContext.cs

```diff
@@ -18,7 +18,7 @@ public class QueueMessageContext : RequestContext
         private Message _requestMessage;
         private Exception _requestMessageException;
         private ReceiveContext _receiveContext;
-        private readonly Lazy<IDictionary<string, object>> _properties = new();
+        private readonly Lazy<IDictionary<string, object>> _properties = new(() => new Dictionary<string, object>());
 
```

Comment:
You could have changed the Lazy field to be a `Lazy<Dictionary<string,object>>` instead so that the default constructor will be used without providing a delegate. The Lazy doesn't need to be the exact same type as the Properties property.

Reply:
done